asserts/database_test.go

```diff
@@ -43,6 +43,7 @@ func Test(t *testing.T) { TestingT(t) }
 
 var _ = Suite(&openSuite{})
 var _ = Suite(&revisionErrorSuite{})
+var _ = Suite(&isUnacceptedUpdateSuite{})
```

Comment:
good catch

Reply:
It also looks like we are a bit inconsistent here where we have the `var _ = Suite(`. For most of the suites it's right next to the `type xxxSuite` but for these we are not. Maybe something for @pedronis to comment on.